app/src/main/java/io/homeassistant/companion/android/settings/SettingsPresenterImpl.kt

```diff
@@ -188,4 +188,10 @@ class SettingsPresenterImpl @Inject constructor(
                     (Integer.parseInt(splitVersion[0]) > 0 || Integer.parseInt(splitVersion[1]) >= 114)
         }
     }
+
+    override fun getNotificationRateLimits(): HashMap<String, *>? {
+        return runBlocking {
+            integrationUseCase.getNotificationRateLimits()!!
```

Comment:
If this service call fails we will crash the app.  We should add some error handling.